homeassistant/components/livisi/translations/de.json

```diff
@@ -0,0 +1,23 @@
+{
```

Comment:
Please remove translations except the default English translation. Use the translations develop script for local testing to get an English translation.